util/dockerfiles/gcn-gpu/Dockerfile

```
@@ -31,6 +31,10 @@ RUN apt -y update && apt -y upgrade && \
     python3-dev python-is-python3 doxygen libboost-all-dev \
     libhdf5-serial-dev python3-pydot libpng-dev libelf-dev pkg-config gdb
 
+# gem5 requires gcc 10+
+RUN apt -y install gcc-10 g++-10 cpp-10
```

Could you update this to:

```suggestion
RUN apt -y update && apt -y install gcc-10 g++-10 cpp-10
```

Otherwise, this change seems fine to me.

---

There is an apt update on the line directly above

---

If those lines are cached, this line can fail. This is a common problem when dealing with Dockerfiles.

---

Make sense.  Should I add that to *all* of the apt installs while I'm updating this file?

---

I don't think that's necessary right now. Adding `apt update -y` is really just a safety. If you encounter problems without it you can just clear the local caches and start from scratch.